translator/source_code/cotrip_translator.py

```diff
@@ -0,0 +1,294 @@
+import json
+from datetime import datetime, timezone, timedelta
+import uuid
+import random
+import string
+import sys, getopt
+from jsonschema import validate
+from jsonschema import ValidationError
+import logging
+from collections import OrderedDict
+import re
+import dateutil.parser
+import translator_shared_library
+
+# Translator
+
+def main():
+    
+    inputfile, outputfile = translator_shared_library.parse_arguments(sys.argv[1:], default_output_file_name = 'cotrip_wzdx_translated_output_message.geojson')
+    if inputfile:
+
+        cotrip_obj = json.loads(open(inputfile).read())
+        wzdx_obj = wzdx_creator(cotrip_obj)
+        location_schema = '../sample files/validation_schema/wzdx_v3.0_feed.json'
+        wzdx_schema = json.loads(open(location_schema).read())
+        if not translator_shared_library.validate_wzdx(wzdx_obj, wzdx_schema):
+            print('validation error more message are printed above')
```

Comment:
If we fail on validation, we should return at this point. Currently, even on validation fail we are opening the output file and writing to it.

Reply:
i changed the print message.should be fixed